troubleshooting/known_issues/issues_with_ncn_health_checks.md

```diff
@@ -0,0 +1,125 @@
+        > This command will not work on the PIT node.
+
+        ```bash
+        velero backup delete <backup> --confirm
+        ```
+
+- `Verify spire-agent is enabled and running`
+
+  - The `spire-agent` service may fail to start on Kubernetes NCNs (all worker and master nodes). In this case, it may log errors
+    (using `journalctl`) similar to `join token does not exist or has already been used`, or the last log entries may contain multiple
+    instances of `systemd[1]: spire-agent.service: Start request repeated too quickly.`. Deleting the `request-ncn-join-token` `daemonset` pod
+    running on the node may clear the issue. Even though the `spire-agent` `systemctl` service on the Kubernetes node should eventually
+    restart cleanly, the user may have to log in to the impacted nodes and restart the service. The following recovery procedure can
+    be run from any Kubernetes node in the cluster.
+
+     1. (`ncn#` or `pit#`) Define the following function
+
+        ```bash
+        function renewncnjoin() {
+            for pod in $(kubectl get pods -n spire |grep request-ncn-join-token | awk '{print $1}'); do
+                if kubectl describe -n spire pods $pod | grep -q "Node:.*$1"; then
+                    echo "Restarting $pod running on $1"
+                    kubectl delete -n spire pod "$pod"
+                fi
+            done
+        }
+        ```
+
+     1. (`ncn#` or `pit#`) Run the function as follows (substituting the name of the impacted NCN):
+
+        ```bash
+        renewncnjoin ncn-xxxx
+        ```
+
+  - The `spire-agent` service may also fail if an NCN was powered off for too long and its tokens expired. If this happens, then delete
+    `/root/spire/agent_svid.der`, `/root/spire/bundle.der`, and `/root/spire/data/svid.key` off the NCN before deleting the
+    `request-ncn-join-token` daemon set pod.
+
+- `cfs-state-reporter service ran successfully`
+
+  - If this test is failing, it could be due to SSL certificate issues on that NCN.
+
+     1. (`ncn#`) Run the following command on the node where the test is failing.
+
+        ```bash
+        systemctl status cfs-state-reporter | grep HTTPSConnectionPool
+        ```
+
+     1. If the previous command gives any output, this indicates possible SSL certificate problems on that NCN.
+
+        - See the [SSL Certificate Validation Issues](ssl_certificate_validation_issues.md) troubleshooting guide.
+
+  - If this test is failing on a storage node, it could be an issue with the node's Spire token. The following procedure may resolve the problem:
+
+     1. (`ncn-m002#`) Run the following script:
+
+        ```bash
+        /opt/cray/platform-utils/spire/fix-spire-on-storage.sh
+        ```
+
+     1. Then re-run the check to see if the problem has been resolved.
+
+- Clock skew test failures
+
+   It can take up to 15 minutes, and sometimes longer, for NCN clocks to synchronize after an upgrade or when a system is brought back up. If a clock skew test
```

Comment:
```suggestion
   It can take up to 15 minutes, and sometimes longer, for NCN clocks to synchronize after an upgrade or when a system is brought back up. If a clock skew test fails, wait for 15 minutes and retry.
```